draft-ietf-tls-esni.md

```diff
@@ -842,6 +857,28 @@ example, if a client-facing server services many backend origin hosts, only one
 of which supports some cipher suite, it may be possible to identify that host
 based on the contents of unencrypted handshake messages.
 
+Beyond these primary security and privacy goals, ECH also aims to hide, to some
```

Comment:
The text below should have a reference to the "don't stick out" doc.

Reply:
Wouldn't hurt!